homeassistant/components/avri/sensor.py

```diff
@@ -0,0 +1,111 @@
+"""Support for Avri waste curbside collection pickup."""
+from datetime import timedelta
+import logging
+
+from avri.api import Avri, AvriException
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, STATE_UNKNOWN
```

Comment:
There is no reason to use `STATE_UNKNOWN`. Return `None` from the state property and the system will mark it as unknkown.

Reply:
Oh yeah right 😅 completely forgot about that. When is `STATE_UNKNOWN` used then?

Reply:
It is used internally when you return `None`. But it's better to let the system swap it out, in case the system ever wants to do other things when a state is unknown (currently it does not)

Reply:
Thanks for the quick answer! Just wanted to know what exactly happens behind the scenes with this particular state.